src/components/view-cards.jsx

```diff
@@ -1,7 +1,42 @@
-import React from 'react';
+import React, { useContext } from 'react';
+import { CardsContext } from './app';
 
 export default class ViewCards extends React.Component {
+  constructor(props) {
+    super(props);
+    this.cards = this.props.cards;
```

Comment:
Avoid storing props like this, you can just access `this.props.cards` in your render method.